posts/2023-08-22-23.0.0.8.adoc

```diff
@@ -0,0 +1,378 @@
+    ...
+    <openidConnectClient pkceCodeChallengeMethod="S256" ... />
+----
+
+If you are using the Social Media Login feature, include configuration similar to the following example:
+
+[source,xml]
+----
+    <featureManager>
+        <feature>socialLogin-1.0</feature>
+    </featureManager>
+    ...
+    <oidcLogin pkceCodeChallengeMethod="S256" ... />
+----
+
+For more information about the configuration options, refer to the docs for the link:{url-prefix}/docs/latest/reference/config/openidConnectClient.html[openidConnectClient element] and the link:{url-prefix}/docs/latest/reference/config/oidcLogin.html[oidcLogin element].
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/25926
+// Contact/Reviewer: Azquelt
+// // // // // // // // 
+[#featureinstall]
+== Ensure sufficient features are installed when using `featureUtility installFeature` command
+When `featureUtility installFeature <featurename>` is used to install a feature on the command line, the feature and all required dependencies are installed.
+
+However, this doesn't guarantee that the feature will start correctly when used with other features in the server, particularly for features that can work with multiple versions of other features. This means that you could list all the features you wanted to use on the command line but find that they didn't all work together because `featureUtility` hadn't installed the right version of every dependency.
+
+To ensure this doesn't happen, running `featureUtility installFeature <featurename>` now installs all versions of any dependencies required by the requested feature, which might result in a larger number of features being installed in some circumstances.
+
+The similar command `featureUtility installServerFeatures <servername>` was not affected by this problem and its behaviour is unchanged. Using `installServerFeatures` is the recommended way to install features as it always installs exactly the minimum set of features needed for the given server configuration.
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+For more details, see:
+
+* link:{url-prefix}/docs/latest/reference/command/featureUtility-installFeature.html[featureUtility installFeature docs]
+* link:{url-prefix}/docs/latest/reference/command/featureUtility-installServerFeatures.html[featureUtility installServerFeatures docs]
+
+// // // // // // // //
+// In the preceding section:
+// Replace TAG_X/SUB_TAG_X with the given tag of your secton from the contents list
+// Replace SUB_FEATURE_TITLE/FEATURE_X_TITLE with the given title from the contents list 
+// Replace FEATURE with the feature name for the server.xml file e.g. mpHealth-1.4
+// Replace LINK with the link for extra information given for the feature
+// Replace LINK_DESCRIPTION with a readable description of the information
+// // // // // // // //
+
```

Comment:
No issue with leaving these in, but I do try to delete them simply to make it easier for reviewers and anyone else to navigate the post contents.
```suggestion
```